internal/app/table_summary.go

```diff
@@ -57,14 +59,19 @@ func (c *consoleWriter) summaryTable(
 	// is almost always the user matching on the wrong package.
 	var passed, notests []summaryRow
 
-	packagePrefix := findCommonPackagePrefix(packages)
+	names := make([]string, 0, len(packages))
+	for _, pkg := range packages {
+		names = append(names, pkg.Summary.Package)
+	}
+	packagePrefix := utils.FindLongestCommonPrefix(names)
 
 	for _, pkg := range packages {
 		elapsed := strconv.FormatFloat(pkg.Summary.Elapsed, 'f', 2, 64) + "s"
```

Comment:
I would have used


```suggestion
		elapsed := pkg.Summary.Elapsed.Truncate(10 * time.Millisecond).String()
```

Reply:
Elapsed is a float64, using `strconv.FormatFloat` seems most appropriate. If it were `time.Time` then truncate would probably make more sense.

Reply:
Oh indeed, I missed that.